glam/urls.py

```diff
@@ -1,29 +1,22 @@
 from django.conf import settings
-from django.urls import path
+from django.urls import path, re_path
 from django.views.generic.base import TemplateView
 
 from .api import views as api_views
-from .auth import views as auth_views
 
 
+spa_view = TemplateView.as_view(
+    template_name="index.html",
+    extra_context={
+        "GA_TRACKING_ID": settings.GA_TRACKING_ID,
+    },
+)
+
 urlpatterns = [
-    path(
-        "",
-        TemplateView.as_view(
-            template_name="index.html",
-            extra_context={
-                "GA_TRACKING_ID": settings.GA_TRACKING_ID,
-            }
-        ),
-        name="index"
-    ),
+    path("", spa_view, name="index"),
     path("api/v1/data/", api_views.aggregations, name="v1-data"),
     path("api/v1/probes/", api_views.probes, name="v1-probes"),
     path("api/v1/probes/random/", api_views.random_probes, name="v1-random-probes"),
     path("api/v1/updates/", api_views.updates, name="v1-updates"),
-    path(
-        "api/v1/front-end-auth-config/",
-        auth_views.front_end_auth_config,
-        name="v1-front-end-auth-config",
-    ),
+    re_path(r"^(?!(?:api|static)(?:$|/)).*$", spa_view, name="catch-all"),
```

Comment:
If we just give a last pattern of `^.*/$` does that work? Whatever pattern doesn't match the above or Django apps would be caught by that and show the SPA view, rather than repeat this twice. I believe `static` should be part of that.